primer3/bindings.py

```diff
@@ -307,30 +334,33 @@ def designPrimers(
 '''
 
 
-def setP3Globals(global_args, misprime_lib=None, mishyb_lib=None):
+def setP3Globals(
+        global_args: Dict[str, Any],
+        misprime_lib: Optional[Dict[str, str]] = None,
```

Comment:
Pretty minor but we support `bytes` object keys and values for the mispriming and mishyb libraries:
https://github.com/libnano/primer3-py/blob/master/primer3/src/primerdesign_helpers.c#L560

Reply:
(might apply in a few other places w/ string I/O -- pretty sure we were consistent in supporting both)

Reply:
I did this correct with the `Union[str, bytes]` flag in other spots.  Let me look at this.

Reply:
Ok, updated to use a `Str_Bytes_T = Union[str, bytes]`